cfme/configure/tasks.py

```diff
@@ -160,3 +160,28 @@ def all_other_tasks(**filter_kwargs):
     Returns: List of dicts.
     """
     return _get_tasks("tasks_all_other", **filter_kwargs)
+
+
+def is_vm_analysis_finished(vm_name, clear_tasks_after_success=False):
```

Comment:
I'll probably make it generic for hosts analysis (datastore doesn't trigger the task AFAIK) in a similar PR for hosts/datastores, stay tuned